python/python_direct/runtime.cpp

```diff
@@ -230,15 +231,43 @@ Examples
           "execute",
           [](FusionExecutorCache& self,
              const py::iterable& iter,
-             std::optional<int64_t> device) {
+             std::optional<int64_t> device,
+             std::vector<std::string> _enable_options,
+             std::vector<std::string> _disable_options) {
             KernelArgumentHolder args = from_pyiterable(iter, device);
+
+            EnableOptionsGuard enable_opt_guard;
+            for (const auto& _enable_option : _enable_options) {
+              std::optional<EnableOption> opt =
+                  stringToEnableOption(_enable_option);
+              NVF_CHECK(
+                  opt.has_value(),
+                  "Unrecognized enable_option: ",
+                  _enable_option);
+              EnableOptionsGuard::getCurOptions().set(opt.value());
+            }
+
+            DisableOptionsGuard disable_opt_guard;
+            for (const auto& _disable_option : _disable_options) {
+              std::optional<DisableOption> opt =
+                  stringToDisableOption(_disable_option);
+              NVF_CHECK(
+                  opt.has_value(),
+                  "Unrecognized disable_option: ",
+                  _disable_option);
+              DisableOptionsGuard::getCurOptions().set(opt.value());
+            }
+
             KernelArgumentHolder outputs = self.runFusionWithInputs(
                 args, std::nullopt, args.getDeviceIndex());
+
             return to_tensor_vector(outputs);
           },
           py::arg("inputs"),
           py::kw_only(),
           py::arg("device") = py::none(),
+          py::arg("_enable_options") = py::none(),
+          py::arg("_disable_options") = py::none(),
```

Comment:
QQ: since the cpp signature is `std::vector`, shouldn't the default be py::list()?